lib/browser/browser.ts

```diff
@@ -426,6 +451,11 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     return accessToken ? accessToken.accessToken : undefined;
   }
 
+  getRefreshToken(): string | undefined {
```

Comment:
so we decided to keep the public method for refresh token?

Reply:
looks like it's neither in readme nor used in code, remove?

Reply:
We kept it because we're exposing it on authState anyway.